src/main.tsx

```diff
@@ -43,7 +43,11 @@ const router = createBrowserRouter([
 				path: 'search',
 				element: <SearchResultsScreen />,
 				loader: searchLoader
-			} 
+			},
+			{
+				path: 'details/:id',
+				element: <ShowDetailsScreen />
+			}
```

Comment:
This should be added [here](https://github.com/Thenlie/Streamability/blob/main/src/__tests__/routes.tsx) as well. We should also create a rough navigation test or have a ticket created to do this. I would just create a simple one in `/screens` for now.